storage-mirror-firestore/POSTINSTALL.md

```diff
@@ -0,0 +1,53 @@
+### See it in action
+
+You can test out this extension right away:
+
+1. Go to your [Cloud Storage dashboard](https://console.firebase.google.com/u/0/project/$%7Bparam:PROJECT_ID%7D/storage).
+2. Upload a file to Cloud storage.
+3. Go to your [Cloud Firestore dashboard](https://console.firebase.google.com/project/$%7Bparam:PROJECT_ID%7D/database/firestore/data)
+4. You can check to see that the corresponding Document was created under `${param:FIRESTORE_ROOT}`.
+
+### Using the extension
+
+Any changes made in Cloud Storage will be reflected in the Firestore mirror. Each delimited path prefix segment in the Object path
+will be separated by `/${param:PREFIXES_SUBCOLLECTION_NAME}/` and the Object itself can be found in 
+the `${param:ITEMS_SUBCOLLECTION_NAME}}` in the Prefix Document representing the parent Prefix.
+
+#### Path Mapping Example
+
+For these Cloud Storage paths, Documents will be generated in the following paths in Firestore.
+
+Configuration Parameters:
+- Firestore Root: `${param:FIRESTORE_ROOT}`
+- Items Subcollection Name: `${param:ITEMS_SUBCOLLECTION_NAME}}`
+- Prefixes Subcollection Name: `/${param:PREFIXES_SUBCOLLECTION_NAME}/`
+
+| Cloud Storage Path                | Firestore Document Path                                                                                                                                                                                                                                                                           |
+| --------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| gs://mybucket/root_photo.jpg      | doc(`${param:FIRESTORE_ROOT}`).collection(`${param:ITEMS_SUBCOLLECTION_NAME}}`).doc('root_photo.jpg')                                                                                                                                                                                             |
```

Comment:
```suggestion
| gs://mybucket/root_photo.jpg      | doc(`${param:FIRESTORE_ROOT}`).collection(`${param:ITEMS_SUBCOLLECTION_NAME}`).doc('root_photo.jpg')                                                                                                                                                                                             |
```

Reply:
@dackers86 did this render correctly in the console? Not sure if we've tried a markdown table in postinstall yet

Reply:
@jhuleatt It did render a table, however text seems unaligned...?

It rendered similar to...

![image](https://user-images.githubusercontent.com/2060661/116586748-b61faf00-a911-11eb-86a3-375b8e85fdc7.png)